Doc/whatsnew/3.12.rst

```diff
@@ -359,7 +359,7 @@ create an interpreter with its own GIL:
    /* The new interpreter is now active in the current thread. */
 
 For further examples how to use the C-API for sub-interpreters with a
-per-interpreter GIL, see :source:`Modules/_xxsubinterpretersmodule.c`.
+per-interpreter GIL, see :source:`Modules/_interpretersmodule.c`.
```

Comment:
No, it is `Modules/_xxsubinterpretersmodule.c` in Python 3.12. Just write:

```
``Modules/_xxsubinterpretersmodule.c``
```

to avoid a link.

Reply:
That makes sense.

```suggestion
per-interpreter GIL, see ``Modules/_xxsubinterpretersmodule.c``.
```